.gitignore

```diff
@@ -58,7 +58,9 @@ standalone/*
 !standalone/*.ts
 standalone/*.d.ts
 ffi/*
+!src/ffi
```

Comment:
This doesn't seem right. If the intention is to stop the line above matching in `/src/` think it's better to change the line above it to:

```
/ffi/*
```

Reply:
this actually works as intended, however cirrus cli wasn’t honouring the git ignore excludes so when it mounted my workspace it was sans src/ffi

will remove

Reply:
Either way, I think the `ffi/*` was definitely meant to be `/ffi/*`